src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,49 @@
 package core.basesyntax;
 
-class Pair {
+import java.util.Objects;
 
+class Pair<T, K> {
+    T first;
```

Comment:
please, add access modifier. I wrote you that at previous review, be careful

Reply:
Sorry about that, my mistake